specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-Preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,1189 @@
+            "$ref" : "#/definitions/EntityRecord"
+          }
+        },
+        "statistics" : {
+          "description" : "(Optional) if showStats=true was specified in the request this field will contain information about the \r\n            document payload."
+        }
+      }
+    },
+    "EntityRecord" : {
+      "type" : "object",
+      "properties" : {
+        "text" : {
+          "type" : "string",
+          "description" : "Entity text as appears in the request."
+        },
+        "type" : {
+          "type" : "string",
+          "description" : "Entity type, such as Person/Location/Org/SSN etc"
+        },
+        "subType" : {
+          "type" : "string",
+          "description" : "Entity sub type, such as Age/Year/TimeRange etc"
+        },
+        "offset" : {
+          "type" : "integer",
+          "format" : "int32",
+          "description" : "Start position (in Unicode characters) for the entity match text."
+        },
+        "length" : {
+          "type" : "integer",
+          "format" : "int32",
+          "description" : "Length (in Unicode characters) for the entity match text."
+        }
+      }
+    },
+    "BioEntitiesBatchResult" : {
+      "type" : "object",
+      "properties" : {
+        "documents" : {
+          "type" : "array",
+          "description" : "Response by document",
+          "readOnly" : true,
+          "items" : {
+            "$ref" : "#/definitions/BioEntitiesBatchResultItem"
+          }
+        },
+        "errors" : {
+          "type" : "array",
+          "description" : "Errors and Warnings by document",
+          "readOnly" : true,
+          "items" : {
+            "$ref" : "#/definitions/ErrorRecord"
+          }
+        },
+        "statistics" : {
+          "description" : "/// <summary>\r\n            (Optional) if showStats=true was specified in the request this field will contain information about the \r\n            request payload.\r\n            </summary>",
+          "readOnly" : true
+        }
+      }
+    },
+    "BioEntitiesBatchResultItem" : {
+      "type" : "object",
+      "properties" : {
+        "id" : {
+          "type" : "string",
+          "description" : "Unique, non-empty document identifier."
+        },
+        "entities" : {
+          "type" : "array",
+          "description" : "Recognized biomedical entities in the document.",
+          "readOnly" : true,
+          "items" : {
+            "$ref" : "#/definitions/EntityRecord"
+          }
+        },
+        "relations" : {
+          "type" : "array",
+          "description" : "Relations of recognized biomedical entities in the document.",
+          "readOnly" : true,
+          "items" : {
+            "$ref" : "#/definitions/RelationRecord"
+          }
+        },
+        "statistics" : {
+          "description" : "(Optional) if showStats=true was specified in the request this field will contain information about the \r\n            document payload."
```

Comment:
We should probably group the statistics item under a new type. Not sure why this only has a description field.